templates/alertmanager.yml.j2

```
@@ -20,11 +21,17 @@ global:
 
 route:
   receiver: 'DevOps'
-  group_by: [alertname, InstanceState]
+  group_by: [alertname, InstanceState, name]
   group_interval: {{ monitoring_alerts_group_message_interval }}
   repeat_interval: {{ monitoring_alerts_repeat_message_interval }}
+  routes: 
+  - receiver: 'null'
+    matchers:
+    - name!~"^(monitoring|dev|test|demo|registry|release|localhost_).*$"
+    - job="container_metrics"
```

I should not be enabled by default. Cause some of our clients won't have such container names.
I suggest
```suggestion
{% if monitoring_except_rules | default([]) | length > 0 %}
  routes: 
  {% for rule in monitoring_except_rules %}
  - receiver: "{{ rule.receiver | default('null') }}"
    matchers:
    {% for matcher in  rule.matchers %}
        - {{ matcher }}
    {% endfor %}
{% endfor %}
```